Sources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTracking</key>
+	<false/>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeOtherDataTypes</string>
```

Comment:
According to [Apple's documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests#4250555), this data type means "Other user contact info" and the description is "Any other information that can be used to contact the user outside the app.". I don't think that's applicable in this case, since the Q42Stats data cannot be used to contact the user.

Reply:
Two data types that I would include in this are:

> `NSPrivacyCollectedDataTypeOtherUsageData`
> Any other data about user activity in the app.
> 
> `NSPrivacyCollectedDataTypeOtherDiagnosticData`
> Any other data collected for the purposes of measuring technical diagnostics related to the app.

Reply:
`NSPrivacyCollectedDataTypeOtherDataTypes` is `Any other data types not mentioned.` I thought that these accessibility items would fall under this category. 

The `Other user contact info` is the `NSPrivacyCollectedDataTypeOtherUserContactInfo` key.

I did not include the `NSPrivacyCollectedDataTypeOtherUsageData` because we don't track user activity. We only track static values and not how the user behaves in the app.

The reason I did not include the `NSPrivacyCollectedDataTypeOtherDiagnosticData` was also for the reason we don't use it for diagnostics aka using this data to debug app issues.

Reply:
Ah, oops! I misread the `NSPrivacyCollectedDataTypeOtherDataTypes` key. You're right!

Regarding `NSPrivacyCollectedDataTypeOtherDiagnosticData`, we could argue that it *is* about diagnostic purposes, since we want to know which features are being used.